README.md

```diff
@@ -218,8 +218,8 @@ libController.on(libController.ItemMouseEnterEventName, function(arg) {
 
 This event is raised when the mouse leaves the range of one library item.
 
-```js
-libController.on("itemMouseLeave", function (searchText) {
+ ```js
+libController.on("itemMouseLeave", function (arg) {
     console.log("Data: " + arg.data);
     console.log("Rect(top, left, bottom, right): " + arg.rect.top + "," + arg.rect.left + "," + arg.rect.bottom + "," + arg.rect.right);
 });
```

Comment:
Can we wrap these long lines so that we don't have horizontal scroll bars on read me